CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ message(STATUS "xtensor-signal v${${PROJECT_NAME}_VERSION}")
 # Dependencies
 # ============
 
-set(xsimd_REQUIRED_VERSION 9.0.1)
+set(xsimd_REQUIRED_VERSION 8.1.0)
```

Comment:
You should keep the 9.0.1 dependency here.

Reply:
I couldn't get xsimd 9.0.0 on conda or vcpkg. Suggestions for sources?

Reply:
Weird, xsimd 9.0.1 is available on conda-forge for windows. What package you get if you run `conda env create -f environment-dev.yml` ?

Reply:
I was running in WSL 2 ands got 8.1.0. Vcpkg is trying to update xsimd to 9.1.0 using the following PR: https://github.com/microsoft/vcpkg/pull/26501. I'm waiting for them to make their update so I can develop in windows using my normal workflow. They seem to be having trouble with apache-arrow...

Reply:
Fixed this!